corehq/apps/hqwebapp/templatetags/menu_tags.py

```diff
@@ -24,7 +24,7 @@ def render(self, context):
         except (ValueError, AttributeError):
             module = None
 
-        tabs = corehq.TABS + getattr(module, 'TABS', ())
+        tabs = getattr(module, 'TABS', corehq.TABS)
```

Comment:
Not related to this, but, this lets the custom domain modules completely override the TABS instead of adding additional ones. Currently no custom domain modules use this, so this is a good opportunity to change this.